### PR TITLE
Handle/errors

### PR DESCRIPTION
**Pull request by tasneembhiri**
### content
- create a Error404 (page not found) 
- create a Error500 (internal server error) 
### testing 
on terminal use the following commands:
- pull origin Handle/Errors.
- npm run dev.
- go to http://localhost:3000/add any end point not exit in route on your browser.

image
image

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Guideline from './components/layouts/Guideline';
 import Checkout from './components/commonComponents/TabsCheckout';
 import NavBar from './components/commonComponents/NavBar';
 import LandingPage from './components/layouts/LandingPage';
-
+import Error404 from './components/layouts/pageNotFound'
```
Comment (okjake) [inline client/src/App.js]: Should use PageNotFound for the filename here, we usually use this casing to indicate that it's a React component

File: server/helpers/errors/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+exports.Error404 = (req, res) => {
```
Comment (okjake) [inline server/helpers/errors/index.js:1]: is this for testing?

Comment (tasneembhiri) [reply server/helpers/errors/index.js:1]: yes, we have consider handling the error at both front and back and here's the back handling version any thoughts? :smile:

**Review by sara219 — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]
🤩🤩🤩🤩🤩🤩

**Pull request merged**